src/utils/string.ts

```diff
@@ -0,0 +1,59 @@
+export function checkEnforcementJSON(test: string): boolean {
```

Comment:
file name is not intuitive, this can maybe be called `checkers` or even still have the name `enforcements` as is kinda what is happening here.

Reply:
Yea, actually there was a file with that name before. The rationale behind moving it to a generic name, was putting all of the string evaluation methods there. At the end, `enforcements` is more specific and accurate to what these methods are doing. So I'll rename it accordingly - also, I will move `parseMethod` to a new file called `parsing.ts` so I can leave only the enforcements there.